tools/check_all_packages.jl

```diff
@@ -5,6 +5,23 @@
 
 using JuliaSyntax, Logging
 
+# like Meta.parseall, but throws
+function parseall(str)
```

Comment:
maybe call this something other than `parseall`, given that we already have `JuliaSyntax.parseall`? It may just be confusing.